initializers/virtual_machines.yml

```diff
@@ -0,0 +1,9 @@
+#- cluster: cluster1
+#  comments: VM1
+#  disk: 200
+#  memory: 4096
+#  name: virtual machine 1
+#  platform: Platform 2
+#  status: 1
```

Comment:
Would it be possible to use the actual word here, i.e. `Active`, `Offline`, `Staged`?

See https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 for how this was made for the `face` property in `devices.yml`.